biztalk/core/feature-pack-create-vsts-token.md

```diff
@@ -66,7 +66,7 @@ Complete [Step 1 - Add Application project and update json](feature-pack-add-app
 > [!NOTE]
 > The build agent is installed on the BizTalk development computer. If using deployment groups, the build agent is installed on all the BizTalk servers you want to deploy to. Also, use these same steps to add a build computer, which might be different than the BizTalk development computer.
 > 
-> Optionally, you can build BizTalk projects using the Azure Pipelines agent pool, instead of the Build Agent. To use the Azure Pipelines agent pool, skip this section, and go to [Step 3: Create the build and release definition](feature-pack-add-build-release-definitions.md).
+> Optionally, you can build BizTalk projects using the Azure Pipelines agent pool, instead of the Build Agent. To use the Azure Pipelines agent pool, skip this section, and go to [Step 3: Create the build and release definition](feature-pack-add-build-definitions.md).
```

Comment:
BizTalk projects can be built with the Azure Pipelines agent pool instead of the Build Agent. To use the Azure Pipelines agent pool, skip this section, and go to [Step 3: Create the build and release definition](feature-pack-add-build-definitions.md).